### PR TITLE
V2.0.0 kowhai ui password

### PR DESCRIPTION
**Pull request by red010b37**
This adds the functionality to save a the UI username and password. I also consolidates the API response and adds send functionality to it.

### CODE REVIEW

File: app/app-config.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
-	"navconf": "$HOME/Library/Application\ Support/NavCoin4/navcoin.conf",
-	"runningNavVersion": "4.1.1",
-	"detectedIp": ""
-}
+	"navconf": "",
+	"runningNavVersion": "",
+	"allowedIps": null,
+	"uiPassword": "$2a$14$TmLKDezfTgOJzMy3YXQ3C.AYj8mD7EEVpimO6NceBiVlr2TM9ds1y"
```
Comment (zanuka) [inline app/app-config.json]: @red010b37 should "uiPassword" be empty, or is this just to verify test?

File: app/boxsetup/setupapi/setupapi.go [hunk 2]
```diff
@@ -10,19 +10,96 @@ import (
...
+	// Protect UI with username and password
+	r.Handle(fmt.Sprintf("/%s/%s/v1/protectui", prefix, nameSpace), middleware.Adapt(protectUIHandler())).Methods("POST")
 
 }
 
+
+
+
+// rangeSetHandler takes the users ip address and saves it to the config as a range
+func protectUIHandler() http.Handler {
```
Comment (zanuka) [inline app/boxsetup/setupapi/setupapi.go]: like this 👍

File: app/app-config.json (hunk 1 above)
Comment (red010b37) [reply app/app-config.json]: This is just to verify and show you what it looks like :)

Comment (zanuka) [reply app/app-config.json]: most excellent

**Pull request merged**